aspnet/mvc/views/forms/sample/final/Views/Demo/RegisterAddress.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@model RegisterAddressViewModel
+
+<form asp-controller="Demo" asp-action="RegisterAddress" method="post">
+    Email:  <input asp-for="Email" /> <br>
+    Password: <input asp-for="Password" /><br>
+    Address: <input asp-for="Address.AddressLine1" /><br />
```

Comment:
Be consistent in all `*.cshtml` files: Use either `<br>` or `<br />`. This mix is a bit weird.